cf-startup.sh

```diff
@@ -0,0 +1,11 @@
+set -e
+set -x
+
+if [ $CF_INSTANCE_INDEX = '0' ]; then
+  python manage.py migrate --fake-initial
+  python manage.py rebuild_index --noinput --remove
+  python manage.py setup_cors
+fi
```

Comment:
Yes, but it lives in atf-eregs, and when I revisited the code this morning, this struck me a simpler way to do the same thing (less indirection)--I said as much in the relevant commit. Do you prefer the management command?